### PR TITLE
Show full GPG commit status on PR commit history

### PR DESCRIPTION
**Pull request by CirnoT**
chrome_2020-05-31_02-27-20

chrome_2020-05-31_02-28-18

### CODE REVIEW

File: templates/repo/commits_list_small.tmpl [hunk 1]
```diff
@@ -31,28 +31,48 @@
...
+								<i class="lock icon"></i>
+								<i class="tiny inverted cog icon centerlock"></i>
+							</i>
+							<img class="ui signature avatar image" src="{{AvatarLink .Verification.SigningEmail}}" />
+						{{end}}
+						</div>
+					{{else}}
+						<i title="{{$.root.i18n.Tr .Verification.Reason}}" class="unlock icon"></i>
+					{{end}}
+				</div>
```
Comment (silverwind) [inline templates/repo/commits_list_small.tmpl]: Would it make sense to extract this badge to its own template file? This is the third occurence of it. [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) 😉

Comment (CirnoT) [reply templates/repo/commits_list_small.tmpl]: Nested templates within nested template within nested template within nested template :>

Will try.

Comment (CirnoT) [reply templates/repo/commits_list_small.tmpl]: Would this be enough?

Comment (silverwind) [reply templates/repo/commits_list_small.tmpl]: Looking good at first glance. Will test later.

**Review by silverwind — APPROVED** [review]

**Review by zeripath — APPROVED** [review]

**Comment by lafriks** [comment]
Make lgtm work

**Pull request merged**